java/c/src/test/python/integration_tests.py

```diff
@@ -183,6 +190,38 @@ def round_trip_reader(self, schema, batches):
     def test_string_array(self):
         self.round_trip_array(lambda: pa.array([None, "a", "bb", "ccc"]))
 
+    def test_stringview_array(self):
```

Comment:
I'd like to see empty arrays, empty strings, nulls in different positions, and actual binary values tested